### ISSUE TITLE
Let MAP_LABEL_OFFSET accept also a  begin|end justification

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#883]
In the example below I had to use a pstext call to print the "years" label and play with the vertical offsets. It would have been easier if we could use **MAP_LABEL_OFFSET=offset[,begin|end]** or **MAP_LABEL_OFFSET=-,begin|end**

![timeline](https://user-images.githubusercontent.com/537321/59262442-1ed6e300-8c37-11e9-9061-7651f4970a59.png)

**Comment by PaulWessel** [comment]
I agree that when one uses graph axis it is often nice to place a label near/below the arrow.  However, it is June 11 and we said we would release GMT 6b on June 15.  I think we need to only spend time on bug fixes (see meca stuff) and warn people against gs 9.27, etc.  We also need to practice the builds and releases stuff. maybe we can do a practice run tomorrow, @leouieda?

**Comment by joa-quim** [comment]
Sure. If i had time I think I could do it myself but really short in time these next days.

(I have a visit from the guy who introduced me GMT 2.4.1. (Suspect not many people understands that importance of that milestone)
